### PR TITLE
DRAFT: Initial files for 2023 TOC Elections

### PR DESCRIPTION
**Pull request by geekygirldawn**
Here is a PR with the initial files required to kick off the Knative 2023 TOC elections.

Before this PR can be merged:
* @jberkus needs to confirm that he can have everything needed to set up Elekto done by May 2.
* @knative/steering-committee needs to officially appoint the election officers. We've proposed @geekygirldawn and @aliok.
* @knative/steering-committee needs to make sure they are OK with the timeline and other details proposed in this PR.

/hold

### CODE REVIEW

**Comment by geekygirldawn** [comment]
Note: I've left @jberkus in the OWNERS file deliberately in case there are any urgent issues, especially related to Elekto, that might come up when the Election Officers are offline (esp due to time zones).

File: elections/2023-TOC/README.md
Comment (aliok) [inline elections/2023-TOC/README.md:51]: Who receives mails to this address? Are we sure that this inbox is working?

File: elections/2023-TOC/README.md
Comment (aliok) [inline elections/2023-TOC/README.md:137]:
```suggestion
Ballots are compiled by Elekto and all candidates are ranked using the [Condorcet]
```

File: elections/2023-TOC/README.md
Comment (aliok) [inline elections/2023-TOC/README.md:153]:
```suggestion
[file a voting exception]: https://elections.knative.dev/app/elections/2023-TOC/exception
[Condorcet]: https://en.wikipedia.org/wiki/Condorcet_method
```

I suggested to create a link for Condorcet above.

**Review by aliok — COMMENTED** [review]
LGTM

I compared files with Knative TOC 2022 election and Kubernetes SC 2022 election files. Suggested some optional minor changes.

I am curious about the elections@knative.team address though.

All good!

File: elections/2023-TOC/README.md
Comment (aliok) [reply elections/2023-TOC/README.md:66]: I've changed all "GitHub"s to "Github" because I noticed you did it in one location.

**Review by Onceagain12 — APPROVED** [review]
Summit

File: elections/2023-TOC/README.md
Comment (geekygirldawn) [reply elections/2023-TOC/README.md:51]: Ah, good question! That alias is configured in groups.yaml and I've updated it there as another commit within this PR. We'll need to wait until the PR is merged and then test it to make sure it's working.

File: elections/2023-TOC/README.md
Comment (geekygirldawn) [reply elections/2023-TOC/README.md:66]: This was a typo in the docs for the last election. Proper capitalization is "GitHub" with a capital H, so I think I've fixed it in most places.

Comment (aliok) [reply elections/2023-TOC/README.md:66]: I did it the other way around, sorry. (Late night PR review)

File: elections/2023-TOC/election_desc.md
Comment (lance) [inline elections/2023-TOC/election_desc.md:6]: Couple of small nits... "WG" is not spelled out anywhere in this document, so I suggest writing it out fully. Also, "individual direction of the project" is a little unclear to me. Here is a suggestion that may be better.
```suggestion
While working groups help shape the direction of individual subprojects, the
```

**Review by lance — COMMENTED** [review]
Looks good overall - thanks for getting this going, @geekygirldawn! I just have a few minor comments

**Comment by geekygirldawn** [comment]
I've addressed all of the feedback on this PR, so we can remove the hold and merge the PR as soon as @jberkus confirms that he's pushed the changes to Elekto that we needed.

**Review by jberkus — CHANGES_REQUESTED** [review]

File: groups/committee-steering/groups.yaml
Comment (geekygirldawn) [reply groups/committee-steering/groups.yaml:26]: We talked about this and would like to leave Josh on the ML for the election officers in case there are any urgent Elekto issues that come up in Pacific time, since Ali and I are both in European time zones.

**Comment by geekygirldawn** [comment]
We've pushed the announcement date for the election to May 3 to give Josh some extra time to set up Elekto and make sure we have everything ready for the announcement.

File: elections/2023-TOC/README.md
Comment (aliok) [inline elections/2023-TOC/README.md:20]:
```suggestion
While working groups help shape the direction of individual subprojects, the
[TOC Charter] covers the technical direction, health of the project
and community as a whole.
```

Copy pasted this better text from the election_desc.md file

**Comment by jberkus** [comment]
/remove-hold
/lgtm

Ready to merge.

**Review by salaboy — APPROVED** [review]

**Pull request merged**
